### PR TITLE
firsto seto o changes

### PR DESCRIPTION
**Pull request by JTGSZ**

### CODE REVIEW

File: F_weebcity/_map_helpers/landmarks.dm [hunk 1]
```diff
@@ -0,0 +1,135 @@
+/* A bunch of defined landmarks with no var-edits mostly so I can place these bitches real fast instead of having to var-edit in a mapping program
+*/
+/obj/effect/landmark/starts //Parent entry in case we need it, also fuck this file and fuck people for expecting me to want to var-edit shitty ass landmarks in 5000 times
+
+
+/obj/effect/landmark/starts/latejoin
```
Comment (AffectedArc07) [inline F_weebcity/_map_helpers/landmarks.dm:6]: Debating making a PR to do this, why do we edit these each time

File: F_weebcity/_maps/testdesert.dmm [hunk 2]
```diff
@@ -0,0 +1,250 @@
+"a" = (/turf/simulated/floor/plating/desert_sand,/area/testdesert)
+"c" = (/obj/effect/landmark/starts/virologist,/turf/simulated/floor/plating/desert_sand,/area/testdesert)
```
Comment (AffectedArc07) [inline F_weebcity/_maps/testdesert.dmm:2]: Should probably be mapmerged

File: code/controllers/subsystem/mapping.dm [hunk 3]
```diff
@@ -23,6 +24,8 @@ SUBSYSTEM_DEF(mapping)
 		fdel("data/next_map.txt") // Remove to avoid the same map existing forever
 	else
 		map_datum = new /datum/map/cyberiad // Assume cyberiad if non-existent
+		*/
+	map_datum = new /datum/map/test_desert
```
Comment (AffectedArc07) [inline code/controllers/subsystem/mapping.dm:28]: You can put this on L25 and have it still work, and just disable the verb side if you want your code to be more modular.

File: code/controllers/subsystem/mapping.dm [hunk 4]
```diff
@@ -51,16 +57,17 @@ SUBSYSTEM_DEF(mapping)
...
 
 
-	// Setup the Z-level linkage
+	// Setup the Z-level linkage 
 	GLOB.space_manager.do_transition_setup()
 
 	// Spawn Lavaland ruins and rivers.
-	log_startup_progress("Populating lavaland...")
+	/* WEEBCITY - JTGSZ CHANGE - Disable lavaland ruin generation to save on init time/server resources unless we are using it.
+	log_startup_progress("Lavaland is disabled SEE: mapping.dm 62")
```
Comment (AffectedArc07) [inline code/controllers/subsystem/mapping.dm:65]: This line won't be called as its inside a comment, so edit may not be that useful.

**Review by AffectedArc07 — COMMENTED** [review]
May as well review it

File: code/controllers/subsystem/mapping.dm (hunk 4 above)
Comment (JTGSZ) [reply code/controllers/subsystem/mapping.dm:65]: thanks for the uhhh review, my bad on this one I wasn't paying much attention and it just picked the main repo

Comment (AffectedArc07) [reply code/controllers/subsystem/mapping.dm:65]: Dont worry about it

**Pull request closed without merging**